x-pack/plugins/security_solution/server/fleet_integration/fleet_integration.test.ts

```diff
@@ -401,7 +405,7 @@ describe('ingest_integration tests ', () => {
       policyConfig.inputs[0]!.config!.policy.value = mockPolicy;
       await expect(() =>
         callback(policyConfig, soClient, esClient, requestContextMock.convertContext(ctx), req)
-      ).rejects.toThrow('Requires Platinum license');
+      ).rejects.toThrow('Gold license does not support this action. Please upgrade your license.');
```

Comment:
Are we changing this existing behaviour?

Reply:
Nope, its just an error message. Previously it was hardcoded [here](https://github.com/elastic/kibana/pull/168113/files#diff-fe231631a50b5450d194aecdd4b763569a053401435aad3d6e3ae25f43c2a9c6L22-R26), with my changes, if user tries to change `global_manifest_version` on platinum license, they would still receive the old message which would be misleading - they are on platinum license already ;). Ive changed it to list a current license they are using and let them know it is not enough.